deepinv/tests/test_loss.py

```
@@ -460,6 +492,11 @@ def test_measplit(device, loss_name, rng):
             eval_n_samples=eval_n_samples,
         )
     elif loss_name == "splitting_eval_split_input_output":
+        if physics_name == "MultiCoilMRI":
+            pytest.skip(
```

Is it raising an error in this case? If yes, maybe it'd be good to check that it actually raises an error rather than just skipping (to avoid having something that works but shouldnt)